Sources/TuistCore/Models/Target.swift

```diff
@@ -126,6 +126,16 @@ public struct Target: Equatable, Hashable, Comparable {
         }
     }
 
+    /// Returns true if the target supports having a headers build phase..
+    public var shouldIncludeHeadersBuildPhase: Bool {
+        switch (platform, product) {
+        case (.iOS, .app):
```

Comment:
if I'm not mistaken, I believe header build phase is only needed for libraries & frameworks on any platform.

```swift
[
   .framework,
   .staticFramework,
   .staticLibrary,
   .dynamicLibrary
]
```

Reply:
I'd say so.

Reply:
updated to only return `true` when one of these product types rather than iOS specific.